site/docs/tools/iceberg/spark.md

```diff
@@ -1,22 +1,9 @@
-# Spark
-
-Nessie can be used with Spark in different ways. These include:
-
-## Iceberg client
-
-Nessie works seamlessly with Iceberg in Spark2 and Spark3. Nessie is implemented as a custom iceberg
-[catalog](http://iceberg.apache.org/custom-catalog/) and therefore supports all features available to any Iceberg
-client. This includes Spark structured streaming, Presto and Flink. See the [iceberg docs](https://iceberg.apache.org)
-for more info.
+# Spark via Iceberg
 
 !!! note
-    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
-    or in a [Jupyter Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb).
+    You can follow along an interactive demo in a [Jupyter Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb).
     
-    Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on
-    [Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-spark-setup.ipynb)
-    and [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-spark-setup.ipynb)
-
+    Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-spark-setup.ipynb)
```

Comment:
The "Binder" link is broken now. (colab/ directory's gone)

Reply:
fixed